dtcli/scripts/dt.py

```diff
@@ -619,6 +624,10 @@ def assemble(source, destination, force):
     if destination.exists() and not force:
         raise click.BadParameter(f"destination {destination} already exists, please try again with --force to proceed irregardless", param_hint="--source")
 
+    if source == destination or is_subdirectory(source, destination):
```

Comment:
It shouldn't be possible for source to equal destination -> right now you can pass this, but if you flip `dir_okay` to False on the `output` it should not come through